# Generated for NaoQi method ALBehaviorManager.getRunningBehaviors
# Get running behaviors

---
# Return running behaviors
string[] value


